setup.cfg

```diff
@@ -29,7 +29,8 @@ install_requires =
     numpy>=1.19
     pyparsing>=2.2
     requests>=2.22
-    roman_datamodels>=0.1.2
+    romanad==0.1.0
```

Comment:
I wouldn't include `romanad` here since `roman_datamodels` takes care of pinning that requirements.

Reply:
Done.